### ISSUE TITLE
Focus management: Draft updates for focus management guidance

### ISSUE THREAD

**Issue by amyleadem** [department-of-veterans-affairs/vets-design-system-documentation#4894]
## Description
Evaluate and update guidance about managing focus on design.va.gov.

## Tasks
- [x] Evaluate current focus management guidance for:
    - [x] Consistency with current best practices 
    - [x] Consistency in language and approach across pages
    - [x] Differences in guidance for SPAs vs static pages
    - [x] Use of plain language
- [x] Draft guidance according to our new recommendations. Guidance should be specific, clear, and written in plain language

## Acceptance Criteria
- [x] We have drafted guidance for improving focus management guidance
- [x] We have a plan to publish the new guidance
